### PR TITLE
Add MySQL REGEXP

### PR DESCRIPTION
**Pull request by SalemCode8**
Adding MySQL REGEXP Operator

### CODE REVIEW

File: docs/find-options.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ userRepository.find({ relations: ["profile", "photos", "videos", "videos.video_a
 * `join` - joins needs to be performed for the entity. Extended version of "relations".
 
 ```typescript
-userRepository.find({ 
+userRepository.find({
```
Comment (klevamane) [inline docs/find-options.md]: This change shouldn' t be added. The same goes for others in this PR

Comment (SalemCode8) [reply docs/find-options.md]: You can remove what you  see isn't necessary

File: src/find-options/FindOperator.ts [hunk 2]
```diff
@@ -112,6 +112,8 @@ export class FindOperator<T> {
                 return `${aliasPath} = ANY(${parameters[0]})`;
             case "isNull":
                 return `${aliasPath} IS NULL`;
+            case "Regexp":
```
Comment (pleerock) [inline src/find-options/FindOperator.ts:115]: to make it consistent, here "Regexp" should be "regexp"

**Comment by pleerock** [comment]
Thank for contribution! This PR needs a test before it can be merged.

**Comment by SalemCode8** [comment]
> Thank for contribution! This PR needs a test before it can be merged.

Yeah, Sure

**Comment by imnotjames** [comment]
We haven't seen any activity on this for a while, there aren't tests, and now have a few conflicts.  I'm going to close this but as soon as you have more time to work on fixing the issues, please open a PR and we'll be gracious to review it again!

For the time being, you can use the new features with `Raw` to pass parameters to that instead!

```
const Regexp = (regex: string) => Raw(alias => `${alias} REGEXP :regex`, { regex });
```

**Pull request closed without merging**